bundle/jsky.app.ot/src/main/java/jsky/app/ot/gemini/editor/targetComponent/EdCompTargetList.java

```diff
@@ -118,22 +118,25 @@ public void textBoxAction(TextBoxWidget tbwe) {
     }
 
     @Override protected void updateEnabledState(boolean enabled) {
-        super.updateEnabledState(enabled);
+        if (enabled != isEnabled()) {
+            setEnabled(enabled);
+            updateEnabledState(getWindow().getComponents(), enabled);
+
+            // Update enabled state for all detail widgets.  The current editor
+            // will have already been updated by the super.updateEnabledState so
+            // update the others.
+            for (final TargetDetailEditor ed : _w.detailEditor.allEditorsJava()) {
+                if (_w.detailEditor.curDetailEdiorJava().forall(cur -> cur != ed)) {
```

Comment:
`curDetailedEdior`: we should probably refactor and fix this horrible spelling error at some point. (Should be Edi__t__or, not Edior.)

Also, is there a reason that we don't do something like:
`final Component[] comps = _w.detailEditor.allEditorsJava().filterNot(_ != _w.detailEditor.curDetailEditorJava()).toArray();`
`updateEnabledState(comps, enabled);`
instead of the `for` loop, or am I missing something?